assets/src/apps/app.tsx

```diff
@@ -5,19 +5,14 @@ import { configureStore } from 'state/store';
 import { Maybe } from 'tsmonad';
```

Comment:
Made a change to registerApplication so we could have a more lightweight application that doesn't get wrapped with a redux store. Used that for both the new VideoPlayer and the existing DarkModeSelector (but didn't do a full inventory of all apps to see if others could exist outside redux)